lms/static/js/student_account/views/LoginView.js

```diff
@@ -29,13 +30,19 @@
                 this.errorMessage = data.thirdPartyAuth.errorMessage || '';
                 this.platformName = data.platformName;
                 this.resetModel = data.resetModel;
+                this.supportURL = data.supportURL;
 
                 this.listenTo( this.model, 'sync', this.saveSuccess );
                 this.listenTo( this.resetModel, 'sync', this.resetEmail );
             },
 
             render: function( html ) {
                 var fields = html || '';
+                var message = gettext('We have sent an email message with password reset instructions to the email address you provided.  If you do not receive this message, {anchorStart}contact technical support{anchorEnd}.'); // jshint ignore:line
```

Comment:
Nit: this isn't the message, it is a message template that will be interpolated into a message. Maybe the simplest thing is to remove the variable and just inline it into the call since it is only used once.